versioned/storage/dynamodb/src/main/java/org/projectnessie/versioned/storage/dynamodb/DynamoDBBackend.java

```diff
@@ -149,7 +157,7 @@ public void eraseRepositories(Set<String> repositoryIds) {
     @SuppressWarnings("resource")
     DynamoDbClient c = client();
 
-    Stream.of(TABLE_REFS, TABLE_OBJS)
+    Stream.of(this.tableRefs, this.tableObjs)
```

Comment:
Here as well as in a few other places, I think `this` is redundant. I know it is not covered by the automated code style but existing code generally refers to fields by simple names.

Reply:
Fixed. I fear I've given myself away a Python dev!